### ISSUE TITLE
Weekly report graph looks a bit strange

### ISSUE THREAD

**Issue by sebastiaanspeck** [range-of-motion/budget#43]
The weekly report looks a bit offset. See the difference in space between month 5 and 6 and the other months. May and June are wider in size

<img width="1213" alt="Schermafbeelding 2019-08-07 om 19 43 14" src="https://user-images.githubusercontent.com/12570668/62645161-f411be00-b94b-11e9-8331-f86ae7eb71e3.png">

**Comment by range-of-motion** [comment]
This is probably due to the labeling. The chart's data is weekly, but the labels are monthly. To determine where to plot labels for the months, we're looking at the last week of each month, and plotting it there.

You can check out `resources/views/reports/weekly_report.blade.php`, as this is where it happens.

**Comment by sebastiaanspeck** [comment]
Maybe change the labelling to Q1, 2, 3 and 4?

**Comment by range-of-motion** [comment]
Seems like an idea, but less specific. Harder to see exactly what happened in—for example—February?

**Comment by sebastiaanspeck** [comment]
Or maybe the view can be toggled between monthly and Q1 etc?

**Comment by range-of-motion** [comment]
Good idea. I'd still prefer to see if we can fix the existing spacing "issue" tho.

**Comment by sebastiaanspeck** [comment]
Agree
